README.md

```diff
@@ -15,6 +15,10 @@ For a quick start just
 [download the pre-built package](https://www.erlang-solutions.com/downloads/download-mongooseim)
 that suits your platform.
 
+Download
+--------
+Feel free to download MongooseIM at: https://www.erlang-solutions.com/downloads/download-mongooseim
```

Comment:
The same link is 5 lines above ;) Though renaming the section to __Download__ might be a good idea.